socorro/processor/rules/breakpad.py

```diff
@@ -359,6 +330,14 @@ def run_stackwalker(self, crash_id, command_path, command_line, status):
                 self.logger.warning(f"{msg} ({crash_id})")
                 output = {}
 
+        else:
+            # Log last 500 characters replacing \n with "\\n"
+            self.logger.error(
+                "MinidumpStackwalkRule: minidump-stackwalk failed: %s %s",
```

Comment:
I should wrap the last part in quotes so it's more obvious when it's an empty string.